### DISCUSSION TITLE
Technologie "Brunnenbau", Erdgas nicht machbar

### DISCUSSION THREAD

**Discussion by Caveman1983** [Taludas/Anno2205-NewFrontiers#12] [General]
Moin, ich bekam die Forschungs-Quest, dass ich "Brunnenbau" im Nexus erforschen soll. Leider nicht machbar...

Desweiteren kann ich keine Off-Shore Plattform für den Erdgasabbau errichten.

**Comment by Taludas** [comment]
Diese Gebäude sind, wie die Quests dir schon versucht haben mitzuteilen, nur durch Technologien aus dem Nexus freischaltbar.
Es sind jeweils die Stufe 3 Technologien eines jeden Zwischenbaums zwischen den zwei Hauptbäumen einer jeden Fachrichtung. So brauchst du für den Brunnenbau die Stufe 3 Technologie zwischen Schwerindustrie und Landwirtschaft. Für Gas die zwischen Energie und Schwerindustrie.
Diese Technologien sind durch freizuschalten durch Aktivierung der anliegenden Stufe 3 Technologien der Hauptbäume. Dafür muss man die entsprechende Anzahl an Laboratorien bauen und die angrenzende Technologie für eine bestimmte Zeit lang aktivieren.
Zur allgemeinen Handhabung des Nexus siehe auch hier: https://anno-2205.fandom.com/de/wiki/Technologie-Nexus
